packages/headless/src/controllers/insight/pager/headless-insight-pager.ts

```diff
@@ -1,10 +1,6 @@
 import {InsightEngine} from '../../../app/insight-engine/insight-engine';
```

Comment:
Could we also adjust `pagerNext` and similar action causes? We see this error quite often on [Barca](https://platformdev.cloud.coveo.com/admin/#/barcasportsmcy01fvu/usage/health/event-problems)

![image](https://github.com/coveo/ui-kit/assets/5565841/a68a3296-4cb8-47e3-96c1-e1a203c455d6)

Reply:
oops, missed a file.
Fixed in cc6834e